src/app/api/userAuth.ts

```
@@ -0,0 +1,71 @@
+	);
+
+	if (!response.ok) {
+		const error = await response.json();
+		throw new Error(error.message);
+	}
+
+	const data = await response.json();
+	const token = data.token;
+	// 토큰 저장
+	localStorage.setItem('authToken', token);
+	console.log(data);
```

ESLint

[error] Unexpected console statement.

(no-console)